### ISSUE TITLE
Dotnet Tool doesn't like F# projects

### ISSUE THREAD

**Issue by BennieCopeland** [security-code-scan/security-code-scan#235]
Running the dotnet tool with a solution that includes a mixture of C# and F# projects fails. An error message is received for each F# project.

```
Cannot open project 'C:\Development\MyProj\src\MyProj.Tools\MyProj.Tools.fsproj' because the file extension '.fsproj' is not associated with a language.
```

I also attempted to exclude the projects with `--excl-proj=**/*.fsproj`, but this also failed to work.

**Comment by JarLob** [comment]
This is a duplicate of #201 See https://github.com/security-code-scan/security-code-scan/issues/201#issuecomment-876307629 for explanation. TL; DR; Use `--ignore-msbuild-errors`.

I don't see other possible fixes except getting list the list of projects from the solution and loading them one by one with filtering applied instead of using `OpenSolutionAsync`. But this would be a reimplementation of `MSBuildWorkspace`.

**Comment by BennieCopeland** [comment]
Using `--ignore-msbuild-errors` worked. Thank you!

**Comment by JarLob** [comment]
FYI, it was implemented in 5.6.4
